src/dmclock/benchmark/README.md

```diff
@@ -0,0 +1,42 @@
+# dmclock benchmarking
+
+**IMPORTANT**: now that K_WAY_HEAP is no longer allowed to have the
```

Comment:
Suggest double backticks ``K_WAY_HEAP``

Why is this value no longer allowed?

Reply:
I'm going to consider this and other changes to the contents of src/dmclock as out of scope for now.  All files under src/dmclock except the ones modified in subsequent commits are actually unchanged, they're just being copied into ceph.git rather than included as a submodule.